app/javascript/controllers/evaluation_criteria_controller.js

```diff
@@ -100,6 +100,18 @@ export default class extends Controller {
     });
   }
 
+  checkPointsOrWeightMax(event) {
+    const input = event.target;
+    const min = input.min;
+    const max = input.max;
+    const value = input.value;
+
+    // If field has a value on blur then check if it's valid
+    if ((value && value < min) || value > max) {
```

Comment:
I'm not sure how to test this but shouldn't the parens surround the min/max?
```suggestion
    if (value && (value < min || value > max)) {
```